### PR TITLE
Fix search reindex cmd

### LINKED ISSUE #1009: `udata search reindex` error
`udata search reindex` w/o argument raises the following error:

```
Traceback (most recent call last):
  File "/Users/alexandre/Developer/Etalab/udata/pyenv/bin/udata", line 11, in <module>
    load_entry_point('udata', 'console_scripts', 'udata')()
  File "/Users/alexandre/Developer/Etalab/udata/udata/commands/__init__.py", line 88, in console_script
    run_manager()
  File "/Users/alexandre/Developer/Etalab/udata/udata/commands/__init__.py", line 84, in run_manager
    manager.run()
  File "/Users/alexandre/Developer/Etalab/udata/pyenv/lib/python2.7/site-packages/flask_script/__init__.py", line 412, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/Users/alexandre/Developer/Etalab/udata/pyenv/lib/python2.7/site-packages/flask_script/__init__.py", line 383, in handle
    res = handle(*args, **config)
  File "/Users/alexandre/Developer/Etalab/udata/pyenv/lib/python2.7/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
  File "/Users/alexandre/Developer/Etalab/udata/udata/search/commands.py", line 90, in reindex
    if adapter.doc_type().lower() == doc_type.lower():
AttributeError: 'NoneType' object has no attribute 'lower'
```

### PR DESCRIPTION
**Pull request by abulte**
Fix #1009

### CODE REVIEW

**Comment by abulte** [comment]
@noirbizarre ok, makes sense.

File: udata/commands/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import print_function
```
Comment (noirbizarre) [inline udata/commands/__init__.py]: Group imports from future

File: udata/commands/__init__.py [hunk 2]
```diff
@@ -81,7 +82,11 @@ def run_manager(config='udata.settings.Defaults'):
     set_logging(app)
     manager.app = app
     register_commands(manager)
-    manager.run()
+    try:
```
Comment (noirbizarre) [inline udata/commands/__init__.py]: Clean error message ? Nice :ok_hand:

**Comment by odtvince** [comment]
LGTM

**Pull request merged**